SeeSharpSnake.csproj

```diff
@@ -46,5 +52,39 @@
     <IlcDisableReflection>true</IlcDisableReflection>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Mode)' == 'CoreRT-CustomPal'">
+    <NoStdLib>true</NoStdLib>
+    <NoConfig>true</NoConfig>
+    <RuntimeMetadataVersion>v4.0.30319</RuntimeMetadataVersion>
+    <Optimize>true</Optimize>
+    <IlcOptimizationPreference>Size</IlcOptimizationPreference>
+    <IlcDisableReflection>true</IlcDisableReflection>
+    <IncludePal>true</IncludePal>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <GenerateTargetFrameworkAttribute>false</GenerateTargetFrameworkAttribute>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(Mode)' == 'CoreRT-CustomPal'">
+    <IlcArg Include="--systemmodule:SeeSharpSnake" />
+    <LinkerArg Include="/subsystem:console /entry:__managed__Main /merge:.modules=.rdata /merge:.pdata=.rdata /DYNAMICBASE:NO /filealign:16 /align:16" />
+  </ItemGroup>
+
+  <Target Name="CustomPal10KByte" BeforeTargets="IlcCompile" AfterTargets="ComputeIlcCompileInputs" Condition="'$(Mode)' == 'CoreRT-CustomPal'">
+    <ItemGroup>
+      <AutoInitializedAssemblies Remove="System.Private.CoreLib" />
+      <AutoInitializedAssemblies Remove="System.Private.DeveloperExperience.Console" />
+      <AutoInitializedAssemblies Remove="System.Private.TypeLoader" />
+      <AutoInitializedAssemblies Remove="System.Private.DisabledReflection" />
+      <AutoInitializedAssemblies Remove="System.Private.Interop" />
+      <IlcReference Remove="@(IlcReference)" />
```

Comment:
Do the references hurt anything? (I would prefer we hardcode as little internal implementation details of the ILCompiler targets files as possible, so if this is only something that shows up in the RSP file, but doesn't affect anything, we can leave it.)

Reply:
System.Private.coreLib added as reference. I now will try to remove that target but my understanding is that it would not work. But I will made an attempt.

Reply:
I double check and both custom targets is required.